### PR TITLE
[ETFE-3821] remove Consignee option on GuarantorArrangerView when MovementScenario doesn't allow a Consignee for this option

### PR DESCRIPTION
**Pull request by james-work-account**
Related: https://github.com/hmrc/emcs-tfe-journey-tests/pull/479

### CODE REVIEW

File: app/models/sections/guarantor/GuarantorArranger.scala [hunk 1]
```diff
@@ -59,10 +59,12 @@ object GuarantorArranger extends Enumerable.Implicits {
...
+    val scenariosWithConsignee: Seq[MovementScenario] = {
+      // Consignee is only available for the following movement scenarios
       UkTaxWarehouse.toList ++ Seq(CertifiedConsignee, TemporaryCertifiedConsignee)
+    }
 
-    val radioItems = if (scenariosWithoutConsignee.contains(movementScenario)) {
+    val radioItems = if (!scenariosWithConsignee.contains(movementScenario)) {
       displayValues.filterNot(_ == Consignee)
     } else {
       displayValues
```
Comment (MJCallahanPage) [inline app/models/sections/guarantor/GuarantorArranger.scala:70]: Would this be more readable to remove the `Not` as it made my head hurt :p 

val radioItems = if (scenariosWithConsignee.contains(movementScenario)) {
      displayValues
    } else {
      displayValues.filterNot(_ == Consignee)
}

**Review by MJCallahanPage — CHANGES_REQUESTED** [review]
One tweak

File: app/models/sections/guarantor/GuarantorArranger.scala (hunk 1 above)
Comment (james-work-account) [reply app/models/sections/guarantor/GuarantorArranger.scala:70]: ... yes

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢

**Pull request merged**